database/migrations/2019_05_16_105124_create_views_table.php

```diff
@@ -0,0 +1,34 @@
+<?php
+
+use Illuminate\Support\Facades\Schema;
+use Illuminate\Database\Schema\Blueprint;
+use Illuminate\Database\Migrations\Migration;
+
+class CreateViewsTable extends Migration
+{
+    /**
+     * Run the migrations.
+     *
+     * @return void
+     */
+    public function up()
+    {
+        Schema::create('views', function (Blueprint $table) {
+            $table->bigIncrements('id');
```

Comment:
dùng big increment à?

Reply:
Vâng, sai gì hả chị, mấy bảng khác em cũng dùng bigIncrements mà

Reply:
https://laraveldaily.com/be-careful-laravel-5-8-added-bigincrements-as-defaults/
đọc qua xem nhé
vì nếu bigIncrement ko cần thiết thì sau này lại lỗi đấy